app/src/main/java/br/com/catbag/gifreduxsample/ui/AnvilRenderComponent.java

```diff
@@ -0,0 +1,9 @@
+package br.com.catbag.gifreduxsample.ui;
+
+/**
+ * Created by raulcca on 10/31/16.
+ */
+
+public interface AnvilRenderComponent {
+    void setAnvilRenderListener(AnvilRenderListener listener);
```

Comment:
Only one observer now, but i think a refactor will be needed to make this code more flexible...i will open a issue to this.